pkg/arvo/sys/vane/gall.hoon

```diff
@@ -630,14 +630,16 @@
         mo-core
       =/  app
         =/  =ship  (slav %p i.t.t.wire)
-        =/  =routes  [disclosing=~ attributing=ship]
+        =/  =routes  [disclosing=~ attributing=[ship /[-.sign-arvo]]]
```

Comment:
I wonder if we'll have problems here with signs that get `%drip`'d through Behn, since using this logic they'll show up as coming from Behn despite coming from some other vane originally.